config/validate_authentication.go

```diff
@@ -16,8 +18,13 @@ func (a *AuthenticationConfig) IsAdmin(key string) bool {
 
 // Validate checks the configuration for specific rules
 func (a *AuthenticationConfig) Validate() error {
-	return validation.ValidateStruct(a,
+	err := validation.ValidateStruct(a,
```

Comment:
Perhaps it's beneficial to check the errors returned by `validation.ValidateStruct` to see if they contain sufficient information. If they do, maybe we should consider disabling the linter for these cases?

WDYT?

Reply:
Errors returned by `validation.ValidateStruct` contains sufficient informations.
![image](https://github.com/bitcoin-sv/spv-wallet/assets/155538368/5af63704-e713-4ff3-a6c6-1fbc7014821d)
added to ignored by wrapcheck